danceradvent/public/articles/2023/15-mason-whats-old-is-new-again.pod

```diff
@@ -0,0 +1,133 @@
+=head1 Dancer2::Plugin::Mason - What's old is new again!
+
+This year, Retro Santa paid us a visit and gave us a new gift of something
+old - an L<HTML::Mason> plugin for Dancer2!
+
+=head2 But Santa, why? WHY?
+
+Mason is a solid templating engine, but it lacks a controller. A
+number of projects have built their own controllers (such as
+L<Request Tracker (RT)|https://bestpractical.com/request-tracker>), and
+others struggle without one. That's where Dancer2 comes in.
+
+L<Dancer2::Template::Mason> is a template engine for HTML::Mason. If
```

Comment:
Dancer2::T::M is more an interface to the engine than an engine itself. 

```suggestion
Well, uh oh no more!
L<Dancer2::Template::Mason> is a brand new interface  for HTML::Mason. If
```

Reply:
Adapter was the word I was really looking for. But I had some momentum and ran with it anyhow.